packages/expo-asset/src/Asset.ts

```diff
@@ -304,4 +304,87 @@ export class Asset {
     }
     return this;
   }
+
+  /**
+   * Returns content of asset as `Uint8Array` from the given URL.
+   *
+   * @platform android
+   * @platform web
+   *
+   * @returns Returns a `Promise` that resolves with the contents of the asset as a `Uint8Array`.
```

Comment:
```suggestion
   * @returns a `Promise` that resolves with the contents of the asset as a `Uint8Array`.
```

The `@returns` automatically adds `Returns ...` before text when parsed in the docs. Let's avoid duplicating it.